From fde22a929e8793c8a21151f22d470fb1d1fb122c Mon Sep 17 00:00:00 2001
From: Meng Wei <wei.meng@intel.com>
Date: Mon, 14 Jan 2019 10:29:24 +0800
Subject: [PATCH 1006/1214] media: intel-ipu4: fix TI960 i2c adapter

for TI960 on the AS_1140 board, i2c adapters are 2, 4.
fix 2nd TI960 i2c apdapter, make it work properly.

Change-Id: Ibcfb52330d295a73e64d98867607b403709b06ec
Tracked-On: PKT-1732
Signed-off-by: Chen Meng J <meng.j.chen@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/platform/intel/ipu4-bxt-p-pdata.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
index 82bebae..53db12e 100644
--- a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
+++ b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
@@ -1238,7 +1238,7 @@ static struct crlmodule_platform_data ov495_pdata = {
 
 #if IS_ENABLED(CONFIG_VIDEO_TI960)
 #define TI960_I2C_ADAPTER	2
-#define TI960_I2C_ADAPTER_2	7
+#define TI960_I2C_ADAPTER_2	4
 #define TI960_LANES	4
 
 static struct ipu_isys_csi2_config ti960_csi2_cfg = {
-- 
2.7.4

